### PR TITLE
feat : Added different types of badges

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Screenshot (329)

> Added badges on avatars

> Added badges on icons

### CODE REVIEW

File: components/Badges/Badges.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+.avatar-badge{
+    height: 1rem;
+    width: 1rem;
+    border-radius: var(--border-radius-f);
+    position: relative;
+   bottom: 1.75rem;
```
Comment (junaidshaikh-js) [inline components/Badges/Badges.css:6]: use code formatting tool for consistent formatiing.

File: components/Badges/Badges.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+.red-badge{
+    background-color: red;
+}
+
+.green-badge{
+   background-color: green;
+}
+
+
+
```
Comment (junaidshaikh-js) [inline components/Badges/Badges.css:19]: unnecessary blank line should not be there

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work!

File: components/Badges/Badges.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    height: 1rem;
+    width: 1rem;
+    border-radius: var(--border-radius-f);
+    position: relative;
+   bottom: 1.75rem;
+   left: 3.75rem;
+   border: 1px solid white;
+}
+.red-badge{
+    background-color: red;
```
Comment (vidhi-thakur) [inline components/Badges/Badges.css:11]: Could use variable color here

**Review by vidhi-thakur — APPROVED** [review]

File: components/Badges/Badges.css (hunk 3 above)
Comment (shuklahimanshu81) [reply components/Badges/Badges.css:11]: Thanks for poiniting.

**Pull request merged**